src/iuse_actor.cpp

```diff
@@ -1181,6 +1181,12 @@ bool firestarter_actor::prep_firestarter_use( const player &p, tripoint &pos )
         p.add_msg_if_player( m_info, _( "There is already a fire." ) );
         return false;
     }
+    // check if there's a fire fuel source spot
+    if( warn_fuel_burn( pos ) ) {
+        if( !query_yn( _( "do you really want to burn your firewood source?" ) ) ) {
```

Comment:
Shouldn't it start with a capital `D`?

Reply:
Dang, you're right.
I'm new to this, can I/you/someone fix this 'in here'? This popup Github just gave me suggests that...but not sure.